taskcluster/examples-mic_vad_streaming-py39.yml

```diff
@@ -0,0 +1,13 @@
+build:
```

Comment:
looks like you will have to rename to `.DISABLED_UNTIL_SCIPY_PY39`, it is failing to run because of no version for Py3.9 yet available